build/cd-build.yml

```
@@ -0,0 +1,39 @@
+# Starter pipeline
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- master
+
```

You'd want to disable this for PRs
```suggestion
pr: none
```